boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java

```diff
@@ -173,6 +167,26 @@ public void testPodAuthors() throws Exception {
         assertEquals(podAuthors, openAPIDevs);
     }
 
+    @Test
+    public void testFromModel() {
```

Comment:
There are no assertions in this test, what is tested exactly?

Reply:
I forgot to add an assertion, i'll add a simple assertion.